packages/@lwc/integration-karma/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+    "extends": "../../../tsconfig.json",
+    "compilerOptions": {
+        "esModuleInterop": true,
+        "noEmit": true,
+        "allowJs": true,
+        "target": "ESNext",
+        "strict": true,
+        "lib": ["ES2015.Promise", "DOM"],
```

Comment:
Why `ES2015.Promise`, rather than something more recent?

Reply:
I haven't given much thought yet to the tsconfig. I started out migrating the spec.js files to spec.ts before I decided on polyfillig jasmine and keeping the tests mostly unchanged. So it is pretty minimal right now.